components/team-members-overview/team-members-overview.js

```diff
@@ -1,19 +1,113 @@
+import React from 'react';
+import Filter from '../filter/filter';
 import TeamMember from '../team-member/team-member';
 import VacancyOverview from '../vacancy-overview/vacancy-overview';
 
-const teamMembersOverview = ({team, vacanciesOverview, vacancies}) => (
-    <div>
-        <ul className="team-members-overview container">
-			{ team.map((teamMember, index) => {
-                if (teamMember.hide) { return; }
-                return <TeamMember key={index} data={teamMember} />
-            })}
-		</ul>
-
-        <VacancyOverview
-            overview={vacanciesOverview}
-            vacancies={vacancies} />
-    </div>
-);
-
-export default teamMembersOverview;
\ No newline at end of file
+
+
+class TeamMembersOverview extends React.Component {
+	constructor(props) {
+		super(props);
+
+		const {team} = props;
+
+		this.roles = {
+			name: 'Roles',
+			data: this.getRoles(team)
```

Comment:
I think data can be an object with `[{value: foo, isActive: false}, {value: bar, isActive: false}]`